server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/TestEndIfDoesNotProduceError.java

```diff
@@ -45,7 +45,8 @@ void test() {
             "1",
             new Diagnostic(
                 new Range(),
-                "Extraneous input 'END'",
+                // TODO: old error message was "Extraneous input 'END'",
+                "No viable alternative at input END",
```

Comment:
These are more or less equivalent from the parser perspective and equaly user-unfriendly, so I'd just keep the new one
```suggestion
                "No viable alternative at input END",
```